src/lib/token_list_gen.ts

```diff
@@ -26,21 +26,31 @@ export interface ArbificationOptions {
 export interface L2ToL1GatewayAddresses {
   [contractAddress: string]: string;
 }
+
+// TODO: read these values from the gateway or a subgraph
```

Comment:
Maybe delete this and create a ticket in notion instead for better visibility.

Reply:
I made the ticket before the TODO tag ;)